fund_store/config/fund_loader_config/FAB/__init__.py

```diff
@@ -51,8 +51,10 @@
             raise ValueError(f"No round_config found in {file}")
         fund_short_name = loader_config["fund_config"]["short_name"]
         round_short_name = loader_config["round_config"]["short_name"]
-        FAB_FUND_ROUND_CONFIGS[fund_short_name] = loader_config["fund_config"]
-        if not FAB_FUND_ROUND_CONFIGS[fund_short_name].get("rounds", None):
+
+        # Ensure the fund exists in the main config
+        if fund_short_name not in FAB_FUND_ROUND_CONFIGS:
+            FAB_FUND_ROUND_CONFIGS[fund_short_name] = loader_config["fund_config"]
             FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"] = {}
```

Comment:
Is this right? Won't it overwrite the rounds if they existed in the fund_config?

Reply:
This block ensures that the fund_short_name is added to FAB_FUND_ROUND_CONFIGS only if it does not already exist. If the fund already exists, this block is skipped.